WebGoatCore/Data/NorthwindContext.cs

```diff
@@ -20,7 +20,7 @@ public static void Initialize(IConfiguration configuration, IHostEnvironment env
             ConnString = builder.ConnectionString;
         }
 
-        public static string ConnString;
+        public static string? ConnString;
```

Comment:
ConnString is used in context which does not accept nulls: 
```
options.UseSqlite(NorthwindContext.ConnString)
[...]
UseSqlite([NotNullAttribute] string connectionString,
```

Reply:
Ok, but isn't it sufficient that we make sure it is not null before it is used in non-null context?

Reply:
But we don't make sure. And I feel that checking it here would be a better place, something like:
`ConnString = builder.ConnectionString ?? throw new SomeException;`

Reply:
Added similar fix to proposed one.